src/opencl_rawsha256_fmt_plug.c

```diff
@@ -165,6 +165,9 @@ static void create_clobj(size_t gws, struct fmt_main *self)
 	uint32_t hash_id_size;
 	size_t mask_cand = 1, mask_gws = 1;
 
+	// Avoid memory leaks.
```

Comment:
I think these comments don't help, and should be dropped. It is implied that if we're freeing resources, we're doing so not to leak them.